src/hooks/websocketListener/useGetWebsocketUrl.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { getWebsocketUrl } from 'utils';
+
+export function useGetWebsocketUrl(apiAddress: string) {
```

Comment:
let's use the same way we do API calls in dapp-core, check getAccountFromApi, for example

Reply:
I already added a "utils" function called: getWebsocketUrl. We have a pair (hook-utils function for getting websocket url)